ext/opcache/tests/preload_loadable_classes_3.phpt

```diff
@@ -9,9 +9,15 @@ opcache.preload={PWD}/preload_loadable_classes_3.inc
 <?php
 require_once('skipif.inc');
 if (PHP_OS_FAMILY == 'Windows') die('skip Preloading is not supported on Windows');
-if (getenv('SKIP_ASAN')) die('xfail Startup failure leak');
 ?>
 --FILE--
-Unreachable
+<?php
+class Foo {
```

Comment:
Related to `ext/opcache/tests/preload_unresolved_prop_type.phpt` - possibly `--FILE--` section unused and only needed for representative purposes?